src/Type/Addon/DateTimeType.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Graphpinator\Type\Addon;
+
+final class DateTimeType extends \Graphpinator\Type\Scalar\ScalarType
+{
+    protected const NAME = 'DateTime';
+    protected const DESCRIPTION = 'DateTime built-in type';
```

Comment:
More advanced description !

Its is not built-in type, but graphpinator add on.